.travis.yml

```diff
@@ -10,11 +13,33 @@ cache:
   directories:
     - $HOME/.gradle/caches/
     - $HOME/.gradle/wrapper/
-    
-script:
-  - wget https://raw.githubusercontent.com/sormuras/bach/76983e2f99497738b6f3e9f86bd42fd3a759a275/install-jdk.sh -O $TRAVIS_BUILD_DIR/install-jdk.sh
-  - bash $TRAVIS_BUILD_DIR/install-jdk.sh --feature 11 -L GPL --target $HOME/openjdk11
-  - ./gradlew build codeCoverageReport -PsystemTestJavaHomes="$JAVA_HOME:$HOME/openjdk11"
-after_success:
-  # upload coverage report
-  - bash <(curl -s https://codecov.io/bash)
\ No newline at end of file
+
+install:
+  - ./gradlew build :inspectit-oce-agent:systemTest
+
+jobs:
+  include:
+    - stage: deploy
+      name: "Code Coverage Reporting"
+      jdk: oraclejdk8
+      install: ./gradlew build codeCoverageReport
+      after_success:
+        # upload coverage report
+        - bash <(curl -s https://codecov.io/bash)
+    - name: "Documentation Deployment"
```

Comment:
Don't we also need the "stage" here? Or does is copy the stage from the previous job?

Reply:
Yes, that's just the name. The stage will be the previous one